### DISCUSSION TITLE
[V3] Nested component with table rows

### DISCUSSION THREAD

**Discussion by sinnbeck** [livewire/livewire#6247] [7. Bugs (archived)] [answered]
I am unsure if this is a bug or I am just doing something wrong.

I have a fullpage component that loads a list of eloquent models and then passes them to a child livewire component.
```php
<table class="w-full">
    <tbody>
    @foreach($applications as $application)
        <livewire:application-row :$application :key="$application->id" >
    @endforeach
    </tbody>
</table>
```
And in the child component I simply have bind it with a public property and then have this in the blade file
```php
<tr>
    <td class="text-red-500">
        <div class="this-becomes-the-new-root">
            <div wire:click="like">Foo</div>
        </div>
    </td>
</tr>
```

The problem still occurs with this basic code. The div with the class "this-becomes-the-new-root" becomes the new root.
![image](https://github.com/livewire/livewire/assets/13980140/559e8c70-04c3-4421-858f-128659617adc)

Here is a quick demo with my actual code, but it happens with my test code as well
![livewire-bug](https://github.com/livewire/livewire/assets/13980140/1a84735d-354a-4b2f-b99b-aa944815abe2)

I think its due to the root being a `<tr>`, instead of a `<div>`. If I use plain divs for everything and move it out of the table, it seems to work.

I am running v3.0.0-beta.7

**Comment by braedenfoster** [comment]
This issue is also occurring for me where the rendering bugs out on update. [Here](https://github.com/braedencrankd/livewire-table-test) is simple reproduction of the issue.

## Steps to reproduce
- Create a livewire component with a table
- Create a nested livewire component that renders a table row
- Update a property on the nested component
- The table row is re-rendered, but the table the row is not in a **tr** tag anymore.

![livewire-table-bug](https://github.com/livewire/livewire/assets/99447151/e1714dd4-6020-46f5-b8a1-8ce1eac001b8)

## Tested on:
Livewire: "^3.0@beta"
Laravel: "^10.10"

**Comment by markjaquith** [comment]
Thanks for this report. I'm also experiencing this exact bug. It worked fine in Livewire 2, and my code has been fully upgraded, so this seems like a bug.

I'm gonna work on making a failing test for it.

**Reply by markjaquith** [reply, marked as answer]
The failing test: https://github.com/livewire/livewire/pull/6296

**Reply by sinnbeck** [reply]
Brilliant! Thanks!

**Reply by braedenfoster** [reply]
Thank you for setting up the test 🙌
